packages/fetchai/skills/erc1155_deploy/skill.yaml

```diff
@@ -0,0 +1,48 @@
+name: erc1155_deploy
+author: fetchai
+version: 0.1.0
+license: Apache-2.0
+fingerprint: {}
+description: "The weather station skill implements the functionality to sell weather data."
+contracts: ['fetchai/erc1155:0.1.0']
+behaviours:
+  service_registration:
+    class_name: ServiceRegistrationBehaviour
+    args:
+      services_interval: 60
+handlers:
+  default:
+    class_name: FIPAHandler
+    args: {}
+  transaction:
+    class_name: TransactionHandler
+    args: {}
+models:
+  strategy:
+    class_name: Strategy
+    args:
+      ledger_id: 'ethereum'
+      is_ledger_tx: True
+      nft: 1
+      ft: 2
+      item_ids: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+      mint_stock: [100, 100, 100, 100, 100, 100, 100, 100, 100, 100]
+      from_supply: 10
+      to_supply: 0
+      value: 0
+      search_schema:
+        attribute_one:
+          name: country
+          type: str
+          is_required: True
+        attribute_two:
+          name: city
+          type: str
+          is_required: True
+      search_data:
+        country: UK
```

Comment:
this seems odd as the search data. I'd prefer something relative to the erc1155 ..